tools/scripts/acr-copy-release.ps1

```diff
@@ -130,6 +132,10 @@ $jobs = @()
 foreach ($Repository in $BuildRepositories) {
 
     $BuildTag = "$($Repository):$($script:ReleaseVersion)"
+    if (![string]::IsNullOrEmpty($script:BuildNamespace) -and `
```

Comment:
@marcschier, I do not see value of `BuildNamespace` being used anywhere else. And here, in the case it is not a already part of `Repository` it is again not used. Is this correct ? Don't we need to append it to repository name somewhere ?

Reply:
No, this is just to pick a namespace from a build registry like industrialiotdev, otherwise the code runs through all namespaces and copies the same tag if available from all causing the wrong repositories (and images) to be picked.